src/main/haskell/kore/test/Test/Kore/Exec.hs

```diff
@@ -280,3 +285,88 @@ applyToNoArgs sort name =
             , symbolOrAliasParams = []
             }
         []
+
+test_execGetExitCode :: TestTree
+test_execGetExitCode =
+    testGroup "execGetExitCode"
+    [ makeTestCase "No getExitCode symbol => ExitSuccess"
```

Comment:
The list here should be indented by four spaces. Please also check your function argument indentation in the `where` block below.